### PR TITLE
CE-3468: Add dismiss button to recent edit banner

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
## Links

* https://wikia-inc.atlassian.net/browse/CE-3468

## Description

Add dismiss button to recent edit banner

## Reviewers

@Wikia/community-engineering @Wikia/x-wing

### CODE REVIEW

**Comment by adamkarminski** [comment]
LGTM

File: front/main/app/templates/components/recent-edit-banner.hbs [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <h3 class="recent-edit-header">
 	{{i18n 'main.title' ns='recent-wiki-activity'}}
+    <button class="recent-edit-close" {{action 'postpone'}}>{{svg 'close' role='img' class='close-icon'}}</button>
```
Comment (bognix) [inline front/main/app/templates/components/recent-edit-banner.hbs]: minor: use tabs not spaces

**Comment by bognix** [comment]
LGTM

**Comment by azarek** [comment]
LGTM

**Pull request merged**